src/index.html

```diff
@@ -2,11 +2,508 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo__image"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#APPLE"
+              class="nav__link is-active"
+            >Apple</a>
+          </li>
+          <li class="nav__item"><a href="#SAMSUNG" class="nav__link">Samsung</a></li>
+
+          <li class="nav__item"><a href="#SMARTPHONES" class="nav__link">Smartphones</a></li>
+
+          <li class="nav__item">
+            <a
+              href="#LAPTOPS&COMPUTERS"
+              class="nav__link"
+              data-qa="nav-hover"
+            >Laptops & computers</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#GADGETS"
+              class="nav__link"
+            >Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#TABLETS"
+              class="nav__link"
+            >Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#PHOTO"
+              class="nav__link"
+            >Photo</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#VIDEO"
+              class="nav__link"
+            >Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="card"
+            class="card__image"
+          >
+
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina
+            5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <h3 class="card__productCode">
+            Product code: 195434
+          </h3>
+
+          <div class="card__starbox">
+            <div class="stars stars--4">
```

Comment:
You also can apply both BEM block and element classes to this element to avoid unnecessary divs